src/Separator/Separator.vala

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env -S vala workbench.vala --pkg gtk4 --pkg gio-2.0
+
+public void main() {
```

Comment:
Please add space between the main function and the parenthesis so the code is formatted by Workbench as expected.
```suggestion
public void main () {
```